src/extension/appcenter/codepush/codepush-sdk/src/cordova/cordova-utils.ts

```diff
@@ -19,35 +22,35 @@ export function getAppVersion(projectRoot?: string): Promise<string> {
       }
 
       const config: any = parsedConfig.widget;
-      resolve(config['$'].version);
+      resolve(config["$"].version);
     });
   });
 }
 
 export function isValidOS(os: string): boolean {
   switch (os.toLowerCase()) {
-    case 'android':
-    case 'ios':
+    case "android":
+    case "ios":
       return true;
     default:
       return false;
   }
 }
 
 export function isValidPlatform(platform: string): boolean {
-  return platform.toLowerCase() === 'cordova';
+  return platform.toLowerCase() === "cordova";
```

Comment:
Does these changes relate with PR goals? I don't see much sense of them to be introduced.